client/my-sites/upgrades/checkout/paypal-payment-box.jsx

```diff
@@ -158,8 +158,14 @@ module.exports = React.createClass( {
 						</button>
 						<SubscriptionText cart={ this.props.cart } />
 					</div>
-					
-					<a href="" className="credit-card-payment-box__switch-link" onClick={ this.handleToggle }>{ this.translate( 'or use a credit card', { context: 'Upgrades: PayPal checkout screen', comment: '"Checkout with PayPal -- or use a credit card"' } ) }</a>
+
+					{ cartValues.isCreditCardPaymentsEnabled( this.props.cart ) &&
+						<a href="" className="credit-card-payment-box__switch-link" onClick={ this.handleToggle }>
+							{ this.translate( 'or use a credit card', {
+								context: 'Upgrades: PayPal checkout screen',
+								comment: '"Checkout with PayPal -- or use a credit card"'
```

Comment:
I don't think we need to surround the comment with double quotes. Maybe it's time to ditch them?